libmscore/rendermidi.cpp

```diff
@@ -154,13 +155,21 @@ void Score::updateChannel()
 //---------------------------------------------------------
 
 static void playNote(EventMap* events, const Note* note, int channel, int pitch,
-   int velo, int onTime, int offTime)
+   int velo, int onTime, int offTime, const Scale& s)
       {
       if (!note->play())
             return;
       velo = note->customizeVelocity(velo);
+
+      double scaleTuning = s.getTuning(note);
+      if (fabs(scaleTuning) > 200) {
+            int step = (int)scaleTuning / 100;
+            pitch += step;
+            scaleTuning += step * 100;
```

Comment:
Shouldn't be `scaleTuning -= step * 100;` ?

For instance, if `scaleTuning` is 250, `step` is increased by 2 (= 200 cent) and `scaleTuning` should be correspondingly be reduced  by 200.